www/front_src/src/Resources/Graph/Performance/timeSeries/index.test.ts

```diff
@@ -221,12 +221,6 @@ describe('timeSeries', () => {
     });
   });
 
-  describe('getMetricValues', () => {
```

Comment:
Did we get rid of the getMetricValues function?

Reply:
Yes because we are using it only in test file. there is no reference in code